uktrade/data-hub-frontend
# Remove Lead ITA

## Description of change

Added the remove _Lead ITA page_ at `/companies/:id/advisers/remove`.

## Test instructions

1. Open a company page by navigating to `/companies` and clicking on the company of your choice
2. Click the _Lead Adviser_ tab
3. If the company already has a Lead ITA assign, you should see a _Remove Lead ITA_ button, if not, click the _Add myself as Lead ITA_ button to add yourself as the Lead ITA and on the subsequent page click the _Add myself as Lead ITA_ button again, which should redirect you back to the company page, where you should now see the _Remove Lead ITA_ button.
4. Click the _Remove Lead ITA_ button
5. You should now see the _Remove Lead ITA_ page conforming to this specification.
6. Clicking the _Cancel_ button should take you back to the company page with the _Lead Adviser_ tab selected, showing the exact state as before, i.e. that the company still has the Lead ITA assigned.
7. Clicking the _Remove Lead ITA_ button should also take you back to the company page with the _Lead Adviser_ tab selected, only this time, you should see a flash message sayning "Lead adviser information updated" and the contentent of the tab shoud say that "This company has no Lead ITA".

## Screenshots

![Screen Shot 2019-12-04 at 9 41 02 AM](https://user-images.githubusercontent.com/2333157/70132734-b3694d80-167c-11ea-9fc1-aa88066c5b89.png)

## Checklist
